pkg/activator/handler/handler.go

```diff
@@ -65,6 +69,16 @@ func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	logger := a.Logger.With(zap.String(logkey.Key, revID.String()))
 
+	tracerRef, err := a.TRGetter(r.Context())
+	if err != nil {
+		// We should never hit this case - tracer.cache should return a noop tracer on error
```

Comment:
If that's the case, can we swallow the error earlier then (by returning a noop tracer) and not return an error in the `TRGetter` at all to solidify this comment?

Reply:
I think we want to log this error (although that can get noisy). It's possible the error is just invalid config (e.g. unparsable enpoint) which would be difficult to debug otherwise.

How about I push this up in to the tracing.HTTP interface as thats really the use case here - I think cache should be a bit more general as IMO its perfectly valid to want to handle the error case from that.